indico/htdocs/js/indico/jquery/ajaxdialog.js

```diff
@@ -184,6 +184,10 @@
                 _.defer(function() {
                     popup.canvas.data('ui-dialog')._focusTabbable();
                 });
+
+                // Re-apply default position after content has been loaded
+                var initialPosition = popup.canvas.dialog('option', 'position');
+                popup.canvas.dialog('option', 'position', initialPosition);
```

Comment:
It's enough to return `true` from this function: https://github.com/indico/indico/blob/master/indico/htdocs/js/indico/Core/Dialogs/Popup.js#L164-L168